scripts/test.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+folders=$(ls data/)
+input_file='input.lyrx'
+output_dir='/usr/share/geoserver/data_dir/styles/ag/'
+for folder in $folders
+do
+    echo 'Converting '$folder
+    output_file=$output_dir$folder'.sld'
+    curl -d @data/$folder/$input_file http://localhost/v1/lyrx2sld/?replaceesri=true -o $output_file
```

Comment:
always prefer the bash `"${}"` synthax in general, the interpreter will be less prone to errors if some strange characters are present in variables or file names and will always cast it as strings if you put the variable in a double quote statement. For example: `"Converting ${folder}"`  or `output_file="${output_dir}${folder}.sld"`